packages/components/src/public/QueryModel/ExportMenu.tsx

```diff
@@ -1,34 +1,106 @@
-import React, { PureComponent, ReactNode } from 'react';
+import React, { FC, memo, PureComponent, ReactNode, useCallback } from 'react';
 import { DropdownButton, MenuItem } from 'react-bootstrap';
 
-import { Set } from 'immutable';
-
 import { EXPORT_TYPES, getQueryModelExportParams, QueryModel, Tip } from '../..';
 
 import { exportRows } from '../../internal/actions';
 
-interface ExportMenuProps {
+export interface ExportMenuProps {
```

Comment:
does this interface and the Impl components below need to be exported? are they used outside of this file?